elevate-cpanel

```diff
@@ -5152,6 +5153,21 @@ EOS
         return;
     }
 
+    sub _fixup_epel_repo ($self) {
+
+        File::Copy::mv(
+            Elevate::Constants::YUM_REPOS_D . '/epel.repo',
```

Comment:
epel is not required on 7. We need to skip if it's not installed.

Reply:
I now test if `epel-release` is installed before proceeding.